controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -308,6 +316,60 @@ func (r *StorageConsumerReconciler) reconcileEnabledPhases() (reconcile.Result,
 
 	} else {
 		r.storageConsumer.Status.State = v1alpha1.StorageConsumerStateDeleting
+		_, hasForceDeleteAnnotation := r.storageConsumer.GetAnnotations()[util.ForceDeletionAnnotationKey]
+
+		consumerOwners := 0
+		for i := range consumerConfigMap.OwnerReferences {
+			if consumerConfigMap.OwnerReferences[i].Kind == "StorageConsumer" {
+				consumerOwners += 1
+			}
+		}
+
+		if consumerOwners <= 1 && hasForceDeleteAnnotation {
+			blockPools := &rookCephv1.CephBlockPoolList{}
+			if err := r.List(r.ctx, blockPools, client.InNamespace(r.namespace)); err != nil {
+				return reconcile.Result{}, err
+			}
+			consumerResources := util.WrapStorageConsumerResourceMap(consumerConfigMap.Data)
+			radosNamespaceName := consumerResources.GetRbdRadosNamespaceName()
+
+			patchPayload := fmt.Sprintf(`{"metadata":{"annotations":{"%s":"true"}}}`, util.RookForceDeletionAnnotationKey)
+			annotationPatch := client.RawPatch(types.MergePatchType, []byte(patchPayload))
```

Comment:
I would prefer to not use hardcoded json payload. You can do something like the following which is typed and when coupled with panic, can guarantee a well formatted result
```	
annotationPatch := client.RawPatch(types.MergePatchType,  util.JsonMustMarshal(
	map[string]any{
		"metadata": map[string]any{
			"annotations": map[string]string{
				util.RookForceDeletionAnnotationKey: strconv.FormatBool(true),
			},
		},
	}),
)
``` 

where `JsonMustMarshal` is a wrapper around `json.Marshal` with a panic for error handling. I beleive we already implemented such a util somewhere